packages/cli/src/commands/start.ts

```
@@ -491,12 +491,63 @@ async function startDashboard(
+/**
+ * bd-8gld (PR #296): Path for the main agent-orchestrator fork.
+ *
+ * AO agents must operate in worktrees, never directly in the main clone.
+ * Guarding here prevents any codepath (spawn, preflight hooks, etc.) from
+ * accidentally writing to the main repo's working tree.
+ *
+ * Configurable via AO_MAIN_REPO env var for non-default installations.
+ * Path comparison uses realpath to canonicalize symlinks, ensuring the guard
+ * works even when project.path contains home-dir aliases or relative paths.
+ */
+import { realpathSync } from "node:fs";
```

ESLint

[error] 'node:fs' import is duplicated.

(no-duplicate-imports)